.gitignore

```diff
@@ -135,3 +135,6 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# VSCode user settings
+.vscode/
```

Comment:
Don't mind this being added here, but it should go in your global ignore: http://egorsmirnov.me/2015/05/04/global-gitignore-file.html